src/components/Button/button.module.css

```diff
@@ -0,0 +1,114 @@
+:root {
```

Comment:
This must be a remainder from a merge with button. This file now exists under Button/src/, you should probably remove this one.